### PR TITLE
Cnv improve registry procedure

### PR DESCRIPTION
**Pull request by apinnick**
Update procedure for creating VDDK image with steps for OpenShift registry

CP for 4.3, 4.4, 4.5

### CODE REVIEW

File: cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc [hunk 1]
```diff
@@ -16,7 +16,46 @@ include::modules/technology-preview.adoc[leveloffset=+1]
...
+
+* You must configure an image registry.
+
+=== Accessing the internal image registry
+
+You can access an {product-title} registry directly, from within the cluster, or externally, by exposing the registry with a route.
+
+.Internal access
+
+You can access the internal image registry xref:../../../registry/accessing-the-registry.html#registry-accessing-directly_accessing-the-registry[directly, from your cluster].
```
Comment (ahardin-rh) [inline cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc:40]: s/.html/.adoc

File: cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc [hunk 2]
```diff
@@ -16,7 +16,46 @@ include::modules/technology-preview.adoc[leveloffset=+1]
...
+
+.External access
+
+You can access the internal image registry externally, xref:../../../registry/securing-exposing-registry.html#registry-exposing-secure-registry-manually_securing-exposing-registry[by exposing the image registry with a route].
```
Comment (ahardin-rh) [inline cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc:44]: s/.html/.adoc

File: cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc [hunk 3]
```diff
@@ -16,7 +16,46 @@ include::modules/technology-preview.adoc[leveloffset=+1]
...
+
+=== Accessing an external image registry
+
+You can access an external image registry from an {product-title} cluster.
+
+.Procedure
+
+. xref:../../../builds/setting-up-trusted-ca.html#configmap-adding-ca_setting-up-trusted-ca[Add certificate authorities] for the external image registry to the {product-title} cluster.
```
Comment (ahardin-rh) [inline cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc:52]: s/.html/.adoc

File: cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc [hunk 4]
```diff
@@ -16,7 +16,46 @@ include::modules/technology-preview.adoc[leveloffset=+1]
...
+
+. Optionally, you can create a xref:../../../openshift_images/managing_images/using-image-pull-secrets.html#images-allow-pods-to-reference-images-from-secure-registries_using-image-pull-secrets[pull secret from your Docker credentials and add it to your service account].
```
Comment (ahardin-rh) [inline cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc:54]: s/.html/.adoc

File: modules/cnv-creating-vddk-image.adoc [hunk 5]
```diff
@@ -3,37 +3,25 @@
 // * cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc
 
 [id="cnv-creating-vddk-image_{context}"]
-= Creating a VDDK image
+= Creating a VMware Virtual Disk Development Kit image
 
-The VMware Virtual Disk Development Kit (VDDK) SDK is used to copy VMware virtual disks.
-
-You can download VDDK, convert it into an image, push the image to a private container repository, and add it to the `v2v-vmware` ConfigMap.
+You can convert the VMware Virtual Disk Development Kit (VDDK) into an image, and push the image to an image registry. You then add the image to the `v2v-vmware` ConfigMap.
```
Comment (ahardin-rh) [inline modules/cnv-creating-vddk-image.adoc:8]: s/You then add/Then, add

**Comment by ahardin-rh** [comment]
@apinnick Just some minor comments from me. Once these changes are squashed into 1 commit, then we should be good to merge. Thanks!

**Comment by apinnick** [comment]
@ahardin-rh  Thanks, Ashley. Changes made and squashed.

**Pull request closed without merging**